scripts/test-sim-backwards.sh

```diff
@@ -47,6 +47,9 @@ PATH=$RELEASE_DIR/bin:$PATH storj-sim -x --host $STORJ_NETWORK_HOST4 network --p
 # run upload part of backward compatibility tests from the lastest release branch
 PATH=$RELEASE_DIR/bin:$PATH storj-sim -x --host $STORJ_NETWORK_HOST4 network test bash "$SCRIPTDIR"/test-backwards.sh upload
 
+# set the segment size lower to make test run faster
+echo client.segment-size: 6.0 MiB >> `storj-sim network env GATEWAY_0_DIR`/config.yaml
```

Comment:
gateway config should have no affect. We are testing uplink.

Reply:
Uplink is using gateway config, because we haven't added automatic uplink config creation yet.